spikeextractors/extraction_tools.py

```diff
@@ -86,14 +85,10 @@ def load_probe_file(recording, probe_file, channel_map=None, channel_groups=None
                 for key_prop, prop_val in cgroup.items():
                     if key_prop == 'channels':
                         ordered_channels = np.concatenate((ordered_channels, prop_val))
-
-            if list(ordered_channels) == recording.get_channel_ids():
-                subrecording = recording
-            else:
-                if not np.all([chan in recording.get_channel_ids() for chan in ordered_channels]) and verbose:
-                    print('Some channel in PRB file are not in original recording')
-                present_ordered_channels = [chan for chan in ordered_channels if chan in recording.get_channel_ids()]
-                subrecording = SubRecordingExtractor(recording, channel_ids=present_ordered_channels)
+            if not np.all([chan in recording.get_channel_ids() for chan in ordered_channels]) and verbose:
+                print('Some channel in PRB file are not in original recording')
```

Comment:
I would raise an error instead no ?

Reply:
No this is perfectly fine. Some channels could be ref and users might want to exclude them from the probe file..I think we discussed about it already!